node-src/lib/findChangedPackageFiles.test.ts

```diff
@@ -141,7 +141,7 @@ describe('findChangedPackageFiles', () => {
   it('considers the file changed if it fails to parse', async () => {
     mockFileContents({
       'package.json': {
-        A: null,
+        A: '',
```

Comment:
This one feels a bit like contortion.  I would say we put this back since it's meant to be the contents of an actual file?